schema.go

```diff
@@ -58,6 +58,7 @@ type Schema struct {
 //	date      | for int32 types use the DATE logical type
 //	timestamp | for int64 types use the TIMESTAMP logical type with, by default, millisecond precision
 //	split     | for float32/float64, use the BYTE_STREAM_SPLIT encoding
+//	id={n}    | where n is int denoting a column field id. Example id=2 for a column with filed id of 2
```

Comment:
How about using the same syntax than we did for other options (e.g., timestamp, decimal)?
```
id(n)
```

Reply:
done 3d5cdec